docs/dev/config/jobservice/config.yaml

```diff
@@ -0,0 +1,14 @@
+grpcPort: 60003
+httpPort: 8090
+subscribeJobSetTime: 10000
+databasePath: "/run/jobservice.db"
```

Comment:
nit: /run seems like a really weird place for this?

Reply:
`/run` is a not-super-common volume mount point, but is definitely used in various distros and virtualized systems - see https://duckduckgo.com/?t=ffab&q=%22%2Frun%22+directory&ia=web . I considered /var/run and /tmp but the former seemed not much improved, and using /tmp, I was nervous that perhaps other things in the container might be doing stuff in there.

Reply:
My objection was mainly because of this use not fitting in the FHS (https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard), but it's in a container and doesn't really matter, which is why I marked it as a nit :D

Reply:
That standard shows the purpose/requirements of `/run` as
```
Run-time variable data: Information about the running system since last boot, e.g., currently logged-in users and running [daemons](https://en.wikipedia.org/wiki/Daemon_(computer_software)). Files under this directory must be either removed or truncated at the beginning of the boot process, but this is not necessary on systems that provide this directory as a [temporary filesystem](https://en.wikipedia.org/wiki/Temporary_filesystem) ([tmpfs](https://en.wikipedia.org/wiki/Tmpfs)).
```
Isn't this exactly the use-case for an ephemeral sqlite3 db file?